docker-compose.yml

```diff
@@ -55,14 +55,26 @@ services:
 
   core:
     # This is a dummy container to build the core image
+    # and document/provide parameters to other compose files
     image: ghcr.io/openrailassociation/osrd-edge/osrd-core:${TAG-dev}
     container_name: osrd-core-dummy
+    depends_on:
+      rabbitmq: {condition: service_healthy}
     build:
       context: core
       dockerfile: Dockerfile
       additional_contexts:
         test_data: tests/data
         static_assets: assets
+    environment:
+      # Should match the reference, see. ./docker/osrdyne.yml
+      CORE_EDITOAST_URL: "http://osrd-editoast"
+      JAVA_TOOL_OPTIONS: "-javaagent:/app/opentelemetry-javaagent.jar"
+      CORE_MONITOR_TYPE: "opentelemetry"
+      OTEL_EXPORTER_OTLP_TRACES_PROTOCOL: "grpc"
+      OTEL_EXPORTER_OTLP_TRACES_ENDPOINT: "http://jaeger:4317"
+      OTEL_METRICS_EXPORTER: "none"
+      OTEL_LOGS_EXPORTER: "none"
```

Comment:
I think directly providing the reference in ./docker/osrdyne.yml is a better idea... Or at least make clear that those are just for documentation, and that `./docker/osrdyne.yml` is not the reference, but the actually used values.

Reply:
Hopefully improved in https://github.com/OpenRailAssociation/osrd/pull/9166/commits/52ffad3e68fc252d8e036c9ec76ba718383749ad#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R70-R71